files/scripts/syncd.sh

```diff
@@ -23,6 +35,12 @@ function startplatform() {
 
         debug "Starting Firmware update procedure"
         /usr/bin/mst start --with_i2cdev
+
+        local -r _MST_DEVICE="$(GetMstDevice)"
+        if [[ "${_MST_DEVICE}" != "${UNKN_MST}" ]]; then
+            /usr/bin/flint -d $_MST_DEVICE --clear_semaphore
```

Comment:
why is there potential semaphore acquired since sdk is not even started?

Reply:
1. We need to clear the semaphore before we are upgrading FW. The FW upgrade is done by SONiC before starting SAI or even `syncd` docker.
2. Architecture and MFT teams approved this change, of unconditionally clear semaphore.